roles/maintenance_10_linux/tasks/main.yml

```diff
@@ -113,6 +113,40 @@
     age: "-{{ 60 * 24 * 7 * 2 }}"  # two weeks
     exclude: "{{ linux_allowed_large_files + linux_additional_allowed_large_files }}"
 
+- <<: *task
```

Comment:
@SaschaD-Adfinis Did I understand the task correctly that the goal is to **replace** 10-034?

If so, the 10-034 tasks above should probably be removed.

Reply:
The code currently labeled as 10-034 aligns with the checklist item 10-037. The task was to rename and enhance the current code, so it aligns to the checklist's naming.
Because there actually is a 10-034 in the checklist, the task included a second part to implement that.

Checklists:
- (10-034) Disk: logrotate: Are there files in /var/log that are not rotated? `find /var/log -type f -size +0 -ls  | sort -k 7 -n`
- (10-037) Disk: Logrotate: Are there large files that point to a too verbose service, service with an issue or should be rotated more often?

Reply:
The task was to :

1. Rename task from 10-034 to 10-037
2. Create a task to identify non-logrotate logs in /var/log